app/js/fullscreen.ts

```diff
@@ -1,9 +1,14 @@
+if (!(document.fullscreenEnabled)) {
+    document.getElementById('toggle-fullscreen').classList.add('hide-toggle-fullscreen');
```

Comment:
if you put this if under the `fullscreenButton` const, you could use that variable instead of selecting the same element from the dom